lib/rubygems/config_file.rb

```diff
@@ -486,4 +471,21 @@ def ==(other) # :nodoc:
   attr_reader :hash
   protected :hash
 
+  private
```

Comment:
Our majority style is to add a blank line after private. I created #2785 to enable the corresponding cop.

Reply:
Done